lua/lvim/plugins.lua

```diff
@@ -158,8 +158,10 @@ local core_plugins = {
     config = function()
       require("lvim.core.which-key").setup()
     end,
-    event = "VeryLazy",
+    event = {"User FileOpened"},
+    keys = {"<leader>"},
     enabled = lvim.builtin.which_key.active,
+    dependencies={"Comment.nvim"}
```

Comment:
```suggestion
```
that's not a good solution, comment.nvim should load when opening a file, it that doesn't work then the autocmd needs fixing